packages/app/src/Feed/TimelineFeed.ts

```diff
@@ -87,19 +91,19 @@ export default function useTimelineFeed(subject: TimelineSubject, options: Timel
       }
     }
     return sub;
-  }, [until, since, options.method, pref, createSub]);
+  }, [until, since, options.method, pref, createSub, subject.type]);
```

Comment:
No need for subject.type here, createSub already has that dependency

Reply:
Removed